# Message describing current environment
Header header

# Robot position
geometry_msgs/Point32 robot_pos

# Dynamic obstacle position
geometry_msgs/Point32 dynamic_obs_pos
